huggan/tensorflow/dcgan/README.md

```diff
@@ -0,0 +1,50 @@
+## Train DCGAN on your custom data
+This folder contains a script to train DCGAN for unconditional image generation, leveraging the Hugging Face ecosystem for processing your data and pushing the model to the Hub.
+
+The script leverages 🤗 Datasets for loading and processing data, and TensorFlow for training the model and 🤗 Hub for hosting it.
+
+## Launching the script
+You can simply run `python train.py` with default parameters. It will download MNIST dataset, preprocess it and train a model on it, will save results of epochs in a local directory and push to 🤗 Hub.
+
+To train on another dataset available on the hub, simply do (for instance):
+
+python train.py --dataset cifar-10
+In case you'd like to tweak the script to your liking, first fork the "community-events" repo (see the button on the top right), then clone it locally:
```

Comment:
```suggestion
```bash
python train.py --dataset cifar-10
```

In case you'd like to tweak the script to your liking, first fork the "community-events" repo (see the button on the top right), then clone it locally:
```